### DISCUSSION TITLE
如果设置pcdn不封禁但限制为1kb传输速度，是否能做到占用其连接资源

### DISCUSSION THREAD

**Discussion by brged** [PBH-BTN/PeerBanHelper#786] [讨论]
如题

**Comment by zhongfly** [comment]
https://docs.pbh-btn.com/docs/downloader/qBittorrentEE#shadowban
qBittorrent-Enhanced-Edition已有类似功能，且pbh已支持

对方也很容易解决，或者为了更快地吸血早就已经会自动断开低速链接。总之，意义不大
